thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -92,6 +93,8 @@
 
     private static final SerializationFormat[] EMPTY_FORMATS = new SerializationFormat[0];
 
+    static AttributeKey<HttpHeaders> HTTP_HEADERS = AttributeKey.valueOf(THttpService.class, "HTTP_HEADERS");
```

Comment:
How about moving this to `ServiceRequestContext`, like `ClientRequestContext` has `HTTP_HEADERS`?